src/renderer/store/modules/profiles.js

```diff
@@ -89,6 +89,33 @@ const actions = {
     commit('setProfileList', profiles)
   },
 
+  async updateSubscriptionDetails({ getters, dispatch }, { channelThumbnailUrl, channelName, channelId }) {
+    if (channelThumbnailUrl === null) {
```

Comment:
What do we handle `undefined` value?

Reply:
fixed in latest commit (option chaining)